src/pages/Wallet/WalletOverview/components/AvailableAssetsTable/AvailableAssetsTable.tsx

```diff
@@ -43,13 +43,13 @@ const AvailableAssetsTable = ({ balances, pooledTickers }: AvailableAssetsTableP
   const { data: vestingData } = useGetVestingData();
   const isMobile = useMediaQuery(theme.breakpoints.down('md'));
 
-  const [showZeroBalances, setShowZeroBalances] = useState(false);
+  const [showAllBalances, setShowAllBalances] = useState(false);
 
   const rows: AvailableAssetsRows[] = useMemo(() => {
     const data = balances ? Object.values(balances) : [];
-    const filteredData = showZeroBalances
+    const filteredData = showAllBalances
       ? data
-      : data.filter((balance) => NATIVE_CURRENCIES.includes(balance.currency) || !balance.transferable.isZero());
+      : data.filter((balance) => NATIVE_CURRENCIES.includes(balance.currency) || balance.currency.ticker === 'USDT');
```

Comment:
@peterslany What should we do here? Specifiying USDT does not look good

Reply:
@danielsimao is there a way to get the tickers for the fee currencies? Dom mentioned that the only currencies we show the zero balance are those which can be used to pay fees, which I hadn't realised.

Reply:
How about
```suggestion
      : data.filter((balance) => [...NATIVE_CURRENCIES.map(({ticker}) => ticker), 'USDT'].includes(balance.currency.ticker);
```

Reply:
If possible I'd like this to be handled in the same we as we handle native currencies, i.e. with a 'FEE_CURRENCIES` const or similar. This is minor though, so no big deal if it can't be done.

Reply:
I dont think it can be done because its a foreign currency @tomjeatt